x/evidence/spec/06_begin_block.md

```diff
@@ -11,12 +11,12 @@ Tendermint blocks can include
 
 ### Equivocation
 
-Currently, the SDK handles two types of evidence inside the ABCI `BeginBlock`:
+The Cosmos SDK handles two types of evidence inside the ABCI `BeginBlock`:
```

Comment:
if the document is live, then "currently" is always assumed 
https://developers.google.com/style/timeless-documentation?hl=en